.github/workflows/ci.yml

```diff
@@ -507,6 +507,74 @@ jobs:
           path: asv_bench/benchmarks.log
         if: failure()
 
+  # This should be migrated to push-to-master.yml before merging!!!
```

Comment:
Since unidist on MPI is fully functional for Modin now, we can leave this job here.

@modin-project/modin-core, thoughts?

Reply:
ok for me. However, we need to think about whether it will be enough to test only `ubuntu` and `python 3.8`?

Reply:
For this PR, I think it is more than enough.

Note: we test only python 3.8 for now in all workflows.

Reply:
fixed